docs/IslandPruning.md

```diff
@@ -5,7 +5,7 @@ true geographic islands are usually disconnected from the main graph. The street
 within the island or continuing from a ferry stop on it works as expected. Even a tiny street network connected to a ferry stop improves
```

Comment:
![Screenshot from 2023-01-27 20-20-36](https://user-images.githubusercontent.com/151346/215178383-d0fe05e5-828d-43f9-843c-fc8fffc08400.png)

The following paragraph in this document also doesn't look well formatted. Can you take a look at this?

Reply:
https://docs.opentripplanner.org/en/dev-2.x/IslandPruning/

Reply:
I tried reformatting the bullet list, hopefully it works.  Is there a way to preview the generated document locally ?

Reply:
You need to install and run `mkdocs`.

```
pip install --user -r docs/requirements.txt
mkdocs serve
```